frontend/packages/aquascope-standalone/styles.scss

```diff
@@ -0,0 +1,336 @@
+  border-style: solid;
+  border-width: 5px;
+  border-radius: 5px;
+  border-color: $color-dark;
+}
+
+@mixin shaded-and-outlined {
+  @include with-border;
+  background-color: $color-light;
+}
+
+/* span configuration */
+
+html,
+body {
+  padding: 3em;
+  height: 100%;
+  font-family: "Verdana", sans-serif;
+  font-size: 18px;
+  background-color: $color-bg;
+}
+
+span.scp {
+  font-variant: small-caps;
+  font-weight: bolder;
+}
+
+div.center {
+  width: 100%;
+  text-align: center;
+}
+
+div.blurb {
+  @include shaded-and-outlined;
+  width: 50%;
+  display: inline-block;
+  margin: auto;
+  padding: 5px;
+  margin-bottom: 4em;
+}
+
+nav {
+  width: 100%;
+  background-color: $color-bg;
+  padding-left: 30px;
+  padding-right: 10px;
+}
+
+.nav-item {
+  display: inline-block;
+  padding: 10px 15px;
+  font-weight: bold;
+  color: $color-lightest;
+}
+
+.nav-item:hover {
+  color: $color-light;
+}
+
+.static-editor {
+  height: 100%;
+  width: 100%;
+  max-height: 700px;
+  max-width: 1200px;
+}
+
+.top-right {
+  z-index: 5;
+  position: absolute;
+  top: 8px;
+  display: flex;
+  flex-direction: column;
+  gap: 0.25em;
+}
+
+.cm-button {
+  @include shaded-and-outlined;
+  padding: 5px 10px;
+  display: inline-block;
+}
+
+.dropdown {
+  /* position: absolute; */
+  display: inline-block;
+}
+
+.dropdown-content {
+  display: none;
+  padding: 12px 6px;
+  position: absolute;
+  background-color: $color-light;
+  min-width: 160px;
+  z-index: 1;
+}
+
+.dropdown-content label {
+  color: black;
+  text-decoration: none;
+  display: block;
+  font-size: 0.8em;
+}
+
+.dropdown-content a:hover {
+  background-color: $color-darkest;
+}
+
+.dropdown:hover .dropdown-content {
+  display: block;
+}
+
+.cm-button:hover,
+.dropdown:hover .btn {
+  background-color: $color-darkest;
+}
+
+/* The container */
+.container {
+  display: block;
+  position: relative;
+  padding-left: 35px;
+  margin-bottom: 12px;
+  cursor: pointer;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
+
+/* Hide the browser's default checkbox */
+.container input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+
+/* Create a custom checkbox */
+.checkmark {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 20px;
+  width: 20px;
+  background-color: $color-dark;
+}
+
+.container:hover input ~ .checkmark {
+  background-color: $color-dark;
+}
+
+.container input:checked ~ .checkmark {
+  background-color: $color-dark;
+}
+
+.checkmark:after {
+  content: "";
+  position: absolute;
+  display: none;
+}
+
+.container input:checked ~ .checkmark:after {
+  display: block;
+}
+
+.container .checkmark:after {
+  left: 9px;
+  top: 5px;
+  width: 5px;
+  height: 10px;
+  border: solid $color-lightest;
+  border-width: 0 3px 3px 0;
+  -webkit-transform: rotate(45deg);
+  -ms-transform: rotate(45deg);
+  transform: rotate(45deg);
+}
+
+.analysis-error-card {
```

Comment:
Should these styles live in `aquascope-standalone`? If we ever want to make the mdbook blocks interactive, we will need to render the error cards in `aquascope-embed` as well.

Reply:
I vote no. Errors in an mdbook version are likely to be reported in a different (hopefully more elegant) way. I've been viewing the standalone package as a sort of "staging area" so we can always move the styles up to the editor later.